pkg/config/config.go

```diff
@@ -730,3 +731,31 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 
 	return sourceOSPath, err
 }
+
+// IsValidArgumentList validates the argument list for `odo config` command
+func IsValidArgumentList(args []string) error {
+	if len(args) == 0 {
+		return nil
+	}
+
+	param, ok := asLocallySupportedParameter(args[0])
+
+	if !ok {
+		return errors.Errorf("the parameter provided : %v, is not present in the supported parameters list.", args[0])
+	}
+	var err error
+	switch param {
+	case "ports", "debugport":
+		err = validation.PortsValidator(args[1])
```

Comment:
What would happen if I provided `odo config set ports` without any value for the ports? Probably would panic due to `args[1]`

Reply:
There's a check for this https://github.com/openshift/odo/pull/3259/files#diff-091ace02b49a46f55f2586cfec89a160R737